<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModelWindow</class>
 <widget class="QWidget" name="ModelWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1132</width>
    <height>673</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>135</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Nom de la figurine</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditName"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>97</width>
     <height>295</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Stats</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>M   </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditM"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>CC </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditCC"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>CT </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditCT"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>F   </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditF"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>E   </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditE"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>I    </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditI"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>A   </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditA"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>PV  </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditPV"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Svg</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditSvg"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Svg Inv</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditSvgInv"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>41</x>
     <y>30</y>
     <width>321</width>
     <height>43</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_11">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Choisissez une race</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboRace">
        <item>
         <property name="text">
          <string>Bretonniens</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Chaos</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Comtes vampires</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Demons du chaos</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Elfes Noirs</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Empire</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Hommes bêtes</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Hommes lézards</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Hauts Elfes</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Nains</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Orcs et Gobelins</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Rois des tombes</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Skavens</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>Type de figurine</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboUnitType">
        <item>
         <property name="text">
          <string>Animal (montures et chars)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Cavalerie</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Char</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Infanterie</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Monstre</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Machine de guerre</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Personnage</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>390</y>
     <width>481</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>360</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Règles spéciales supplémentaires</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>460</y>
     <width>121</width>
     <height>128</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string>Taille du socle</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_15">
      <item>
       <widget class="QLabel" name="label_20">
        <property name="text">
         <string>Largeur</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_13">
        <item>
         <widget class="QSpinBox" name="spinWidth">
          <property name="maximum">
           <number>999</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>mm</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_14">
      <item>
       <widget class="QLabel" name="label_19">
        <property name="text">
         <string>Longueur</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <item>
         <widget class="QSpinBox" name="spinLength">
          <property name="maximum">
           <number>999</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>mm</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QLabel" name="label_21">
        <property name="text">
         <string>Puissance d'unité</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>230</y>
     <width>141</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSave">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>620</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Sauver Figurine</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>620</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Annuler</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>280</y>
     <width>331</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <widget class="QLabel" name="label_22">
      <property name="text">
       <string>Image</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_16">
      <item>
       <widget class="QLineEdit" name="lineEditImage"/>
      </item>
      <item>
       <widget class="QToolButton" name="toolButtonImage">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
